src/api/v4/layer/event-types.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Do we go with these enums for events?

If affirmative, we already have something in `src/api/v4/events.js`.

I don't know if we want to group all events in one place or disseminated through the codebase.

Reply:
We should agree about this... It should be consistent across the API. We're not doing this for dataview events.